docs/join-the-community.md

```diff
@@ -0,0 +1,19 @@
+# Join the community
+
+## Chat
+
+If you need support, want to contribute or just want to say hi, join us at the [source{d} community Slack](https://join.slack.com/t/sourced-community/shared_invite/enQtMjc4Njk5MzEyNzM2LTFjNzY4NjEwZGEwMzRiNTM4MzRlMzQ4MmIzZjkwZmZlM2NjODUxZmJjNDI1OTcxNDAyMmZlNmFjODZlNTg0YWM). We hang out in the #general channel.
+
+## Contributing
+
+You can start contributing in many ways
```

Comment:
Add `:` to the end of the phrase. Also delete all final `.` in all items as the other lists don't have them.